wagtail/admin/tests/test_page_chooser.py

```diff
@@ -257,13 +257,12 @@ def test_admin_display_title_breadcrumb(self):
 
         # Look for a link element in the breadcrumbs with the admin title
         expected = """
-            <li class="breadcrumb-item">
-                <a href="/admin/choose-page/{page_id}/?" class="breadcrumb-link navigate-pages">{page_title}
-                    <svg class="icon icon-arrow-right arrow_right_icon" aria-hidden="true">
-                        <use href="#icon-arrow-right"></use>
-                    </svg>
-                </a>
-            </li>
+            <li class="w-h-full w-flex w-items-center w-overflow-hidden w-transition w-duration-300 w-whitespace-nowrap w-flex-shrink-0 w-font-bold " data-breadcrumb-item="">
+                                <a class="w-flex w-items-center w-h-full w-text-primary w-px-0.5 w-text-14 w-no-underline w-outline-offset-inside hover:w-underline hover:w-text-primary w-h-full" href="/admin/choose-page/{page_id}/?">
+                                    {page_title}
+                                </a>
+                                <svg class="icon icon-arrow-right w-w-4 w-h-4 w-ml-3" aria-hidden="true"><use href="#icon-arrow-right"></use></svg>
+                            </li>
```

Comment:
try to align the spacing correctly so it is nested well - makes it easier to read.